### PR TITLE
NO-ISSUE - don't give up during sno log collection

### PR DESCRIPTION
**Pull request by omertuc**

### CODE REVIEW

File: discovery-infra/bootstrap_in_place.py [hunk 1]
```diff
@@ -151,6 +153,37 @@ def gather_sosreport_data(node):
     node.download_file("/tmp/sosreport.tar.bz2", IBIP_DIR)
 
 
+# noinspection PyBroadException
+def log_collection(controller, vm_ip):
+    etype, _value, _tb = sys.exc_info()
+
+    logging.info(f"Collecting logs after a {('failed', 'successful')[etype is None]} installation")
```
Comment (eranco74) [inline discovery-infra/bootstrap_in_place.py:160]: Didn't know this syntax

**Review by eranco74 — COMMENTED** [review]
/lgtm

File: discovery-infra/bootstrap_in_place.py (hunk 1 above)
Comment (omertuc) [reply discovery-infra/bootstrap_in_place.py:160]: i'm actually abusing the fact that bool gets converted to 0/1 and using that as an index into the tuple.... it's very ugly but very readable

**Pull request merged**